src/handlePreviewToggles.ts

```diff
@@ -1,36 +1,39 @@
 "use strict";
 
-import { unset, has } from "lodash";
 import { bold, red } from "colorette";
 
-import { configstore } from "./configstore";
-import { previews } from "./previews";
+import * as experiments from "./experiments";
 
-function _errorOut(name?: string) {
-  console.log(bold(red("Error:")), "Did not recognize preview feature", bold(name || ""));
+function errorOut(name?: string): void {
```

Comment:
`name` is never used. Remove or call `errorOut` with a name?

Reply:
How about instead I just start passing name in? That seems like it'll provide better value